docs/spelling_wordlist.txt

```diff
@@ -58,3 +58,5 @@ Walkthrough
 walkthrough
 Workflow
 webhook
+Kopia
```

Comment:
I don't think we need `Kopia` here. Can you please verify.

Reply:
Yes @viveksinghggits , without it, `make docs` command is giving some error.